Admin/AdminInterface.php

```diff
@@ -266,18 +266,6 @@ public function getParentFieldDescription();
     public function hasParentFieldDescription();
 
     /**
-     * translate a message id.
-     *
-     * @param string $id
-     * @param array  $parameters
-     * @param null   $domain
-     * @param null   $locale
-     *
-     * @return string the translated string
-     */
-    public function trans($id, array $parameters = array(), $domain = null, $locale = null);
-
-    /**
```

Comment:
This is a BC break. Any personal implementation of AdminInterface will break if you do this.